Assets/Scripts/Player/PlayerController.cs

```diff
@@ -262,6 +264,24 @@ public class PlayerController : MonoBehaviour {
 				crawlingSound.Stop();
 			}
 		}
+
+		if (isGrounded && transformation.State == TransformationState.Wolf && Mathf.Abs(velocity.x) > 0.1f) {
+			if (!wolfRunningSound.isPlaying) {
+				wolfRunningSound.Play();
+			}
+		}
+		else {
+			wolfRunningSound.Stop();
+		}
+
+		if (isGrounded && transformation.State == TransformationState.Human && Mathf.Abs(velocity.x) > 0.1f) {
+			if (!humanRunningSound.isPlaying) {
+				humanRunningSound.Play();
+			}
+		}
+		else {
+			humanRunningSound.Stop();
+		}
```

Comment:
Perhaps using xInput is better to use. Since velocity didn't really fix the problem of turning fast. Maybe using resume/pause instead of play/stop would solve this alongside falling down a small ledge.

Reply:
applied it - it isn't perfect, but definitively better.. though I am reconsidering simply having it Play/Stop with xInput, might be weird if spammed, but otherwise felt best..

Reply:
You're using `Play`, not `Resume` in your last commit (10588d9). Play restarts the sound from the beginning.

Reply:
But there is no "Resume" ?

Reply:
Oh, my bad! It's called `UnPause`. Also, you might have to call `Play` atleast once before using it. Not sure.